disasterinfosite/load.py

```diff
@@ -1,14 +1,71 @@
+import math
 import os
+from django.contrib.gis.gdal import GDALRaster
+from django.contrib.gis.geos import Polygon
 from django.contrib.gis.utils import LayerMapping
 
 
+# If a raster's height or width is > this threshold, tile it
+# In theory this can be up to INT_MAX * 2, but that runs into memory constraints
+rasterTileMaxDimension = 5000
```

Comment:
Something just occurred to me about this constant.  It's currently set to a 5000 because 10,000 fails and I intuitively thought that having a larger number of smaller tiles would be bad for performance.  But I now think that smaller tiles might actually be better because the point-in-raster search uses precomputed bounding boxes to not even bother looking into tiles which won't get a hit.  And loading a 5000x5000 tile is slower than looking at more bboxes.  Is this worth experimenting with?  (my hunch is only if we notice speed being a problem)

Reply:
Yeah, I think so, actually. Databases are pretty good for handing a lot of little things (better than they are at handling a few big things), so if we think it might speed up queries, it's worth looking into.

Reply:
OK!  I'll do some empirical testing in that case, with the new files so we have multiple large rasters.